mbofui/package.json

```diff
@@ -0,0 +1,17 @@
+
+{
+  "name": "Ocellus",
```

Comment:
As mentioned above, I thought we were keeping the name of the application here. Why is it hard coded in index.html?